pkg/kubernetes/scripts/virtual-machines/components/GeneralTab.jsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// @flow
+import React from 'react'
+import { gettext as _ } from 'cockpit'
+
+import type { Vm, VmMessages } from '../types.jsx'
+import { getPairs, NODE_LABEL } from '../utils.jsx'
+import VmMessage from './VmMessage.jsx';
```

Comment:
Please be consistent wrt. using a semicolon or not; most other code in cockpit seems to use them. Same for `const` declarations here, it's a bit irritating to see them not use `;` while everything else does (has to).

Reply:
Sure. I think we should use them to be consistent with the rest of cockpit.
If so, we need to add semi-colons to the rest of fresh kubevirt-related code comming from other patches.

IMO, it would be great to set linter rules for code-style. Do you think it's worth to try in a follow-up?

Reply:
Absolutely, if you already know how to configure it. Even a failing PR would be nice as a starter, then we can help fixing pages.

Reply:
For the beginning, I can implement it as a new webpack script (defined within package.json). It will check selected directories.
Once its working somehow (shall be simple), we can discuss rules, directories to be included and how to enforce within other tests.
@martinpitt , can we leave this semi-colon-mess as a feed for this follow-up? :-)

Reply:
Yes, sure. not important, just triggered my OCD :-)